lib/event_sourcery/event_processing/event_stream_processor.rb

```diff
@@ -40,7 +40,7 @@ def process(event)
           end
           @_event = nil
         rescue
-          raise EventProcessingError.new(event)
+          raise EventProcessingError.new(event), "Processor: #{processor_name.to_s}\nEvent: #{event.inspect}"
```

Comment:
Maybe we could just forward the message from the rescued error to the raised error here. 🤔